### DISCUSSION TITLE
Git rebase -> fatal: invalid upstream

### DISCUSSION THREAD

**Discussion by patriceguyot** [alshedivat/al-folio#692] [Q&A] [answered]
Hi,
Thank you very much for this great code! 

I try to upgrade, but went I hit the third command you mention:
`git rebase upstream/v0.5.0`
it returns an error:
`fatal: invalid upstream 'upstream/v0.5.0'`

Do you know how to handle this? 
Thanks!

**Comment by rohandebsarkar** [comment, marked as answer]
I think you have not added the remote upstream in your git yet.

run `git remote add upstream https://github.com/alshedivat/al-folio.git`

**Reply by patriceguyot** [reply]
Thanks for your answer!
Unfortunately, the remote upstream was already added (see command below).
    
    >git remote
    origin
    upstream

Any other idea?

**Reply by rohandebsarkar** [reply]
Try `git rebase v0.5.0`.

I think there is an error in the README documentation/

**Reply by patriceguyot** [reply]
Ok, thanks, it works now!
As stated in the documentation... it seems quite complicated to resolve the conflicts!
Thanks a lot for the tip anyway!
